terraform/code.gov.tf

```diff
@@ -42,6 +42,18 @@ resource "aws_route53_record" "staging_code_gov_a" {
   }
 }
 
+resource "aws_route53_record" "developers_code_gov_a" {
```

Comment:
second string should be "code_gov_developers_code_gov_a" I think

Reply:
@froi once this is fixed I can 👍 and merge

Reply:
@wslack I followed the configuration for `staging.code.gov` on [line 33](https://github.com/18F/dns/pull/220/files#diff-0e0045f50c52bd5e960af32355a1c8f5R33). Should that be changed as well to keep it consistent?